distribution/docs/src/main/resources/content/_using/attribute-editor.adoc

```diff
@@ -0,0 +1,21 @@
+:title: Using the Attribute Editor
+:type: subUsing
+:status: published
+:parent: Pan to a Location
+:summary: Using the Attribute Editor
+:order: 00
+
+== {title}
+
+
+${catalog-ui} provides an upload editor form which allows users to customize the metadata of their
```

Comment:
❔ are `upload editor` and `attribute editor` the same thing? Seems like this should be `attribute editor`?

Reply:
went with attribute editor